src/components/biggive-form-field-select/biggive-form-field-select.tsx

```diff
@@ -7,44 +7,43 @@ import { Component, Prop, h, Event, Element, EventEmitter, Listen } from '@stenc
 })
 export class BiggiveFormFieldSelect {
   @Element() el: HTMLDivElement;
-  /**
-   * This event `doChange` event is emitted and propogates to the parent
-   * component which handles it
-   */
-  @Event({
-    eventName: 'doSelectChange',
-    composed: true,
-    cancelable: true,
-    bubbles: true,
-  })
-  doSelectChange: EventEmitter<object>;
 
+  @Prop()
+  onSelectionChange: (value: string) => void;
   /**
    * Displayed as 'eyebrow' label over the top border of the box.
    */
   @Prop() prompt!: string | null;
 
-  @Prop() selectedValue: string | null;
-  @Prop() selectedLabel: string | null;
+  @Prop({ mutable: true }) selectedValue: string | null;
+  @Prop({ mutable: true }) selectedLabel: string | null;
```

Comment:
I think the existing code was generating warnings in the console because this prop value is mutated but it wasn't marked as mutable before.